docs/authentication.md

```diff
@@ -103,4 +109,7 @@ var sdk = new BoxSDK({
 	}
 });
 var adminClient = sdk.getAppAuthClient('enterprise', 'YOUR-ENTERPRISE-ID');
+// Machine to machine
+var enterpriseApp = sdk.getEnterpriseAppAuthTokens('YOUR-ENTERPRISE-ID', callback);
+var userApp = sdk.getAppUserTokens('APP-USER-ID', callback)
```

Comment:
Why would the user want to access these methods directly?  It seems more likely to me that we would want to document the process for getting a client (see line 111), since that's how they would make calls to the API.